pyvista/_vtk.py

```diff
@@ -111,16 +111,34 @@
         vtkFloatArray,
         vtkIdList,
         vtkIdTypeArray,
+        vtkIntArray,
+        vtkLongArray,
+        vtkLongLongArray,
         vtkLookupTable,
         vtkOutputWindow,
         vtkPoints,
+        vtkShortArray,
         vtkSignedCharArray,
+        vtkSparseArray,
         vtkStringArray,
         vtkStringOutputWindow,
+        vtkTypedArray,
+        vtkTypeFloat32Array,
+        vtkTypeFloat64Array,
+        vtkTypeInt8Array,
+        vtkTypeInt16Array,
         vtkTypeInt32Array,
         vtkTypeInt64Array,
+        vtkTypeUInt8Array,
+        vtkTypeUInt16Array,
         vtkTypeUInt32Array,
+        vtkTypeUInt64Array,
+        vtkUnicodeStringArray,
         vtkUnsignedCharArray,
+        vtkUnsignedIntArray,
+        vtkUnsignedLongArray,
+        vtkUnsignedLongLongArray,
+        vtkUnsignedShortArray,
```

Comment:
If we're no longer relying on checking for these new array subclasses in `_vtk`, should we revert these added imports?

Reply:
Done.